### PR TITLE
[WIP] Contributing toxhelp

### PR DESCRIPTION
**Pull request by flv0**
please review

[Review on Reviewable](https://reviewable.io/reviews/the-compiler/qutebrowser/1300)

### CODE REVIEW

File: CONTRIBUTING.asciidoc [hunk 1]
```diff
@@ -89,19 +89,38 @@ Checkers
...
-* `scripts/misc_checks.py` which checks for the following things:
+Currently, following tox environments are available:
+
+* Unit tests using https://www.pytest.org[pytest]:
+  - `py34`: Run pytest for python-3.4.
+  - `py35`: Run pytest for python-3.5.
+  - `py34-cov`: Run pytest for python-3.4 with code coverage report.
+  - `py35-cov`: Run pytest for python-3.4 with code coverage report.
+  - `unittests-frozen`: ?
+  - `unittests-nodisp`: ?
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: Not sure if they should be mentioned at all? If so, `frozen` would be "Run pytest on tests frozen by cx_Freeze (mostly useful on Windows)" and `nodisp` would be "Run tests without `DISPLAY` set (on Linux)" or something.

File: CONTRIBUTING.asciidoc [hunk 2]
```diff
@@ -89,19 +89,38 @@ Checkers
...
-* https://pypi.python.org/pypi/pyroma/[pyroma]
-* https://github.com/mgedmin/check-manifest[check-manifest]
-* `scripts/misc_checks.py` which checks for the following things:
+Currently, following tox environments are available:
+
+* Unit tests using https://www.pytest.org[pytest]:
+  - `py34`: Run pytest for python-3.4.
+  - `py35`: Run pytest for python-3.5.
+  - `py34-cov`: Run pytest for python-3.4 with code coverage report.
+  - `py35-cov`: Run pytest for python-3.4 with code coverage report.
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: Nitpick: I think `python 3.4` etc. without the dashes would look better

File: CONTRIBUTING.asciidoc [hunk 3]
```diff
@@ -89,19 +89,38 @@ Checkers
...
-* https://pypi.python.org/pypi/pep8[pep8] via https://pypi.python.org/pypi/pytest-pep8[pytest-pep8]
-* https://pypi.python.org/pypi/mccabe[mccabe] via https://pypi.python.org/pypi/pytest-mccabe[pytest-mccabe]
-* https://github.com/GreenSteam/pep257/[pep257]
-* http://pylint.org/[pylint]
-* https://pypi.python.org/pypi/pyroma/[pyroma]
-* https://github.com/mgedmin/check-manifest[check-manifest]
-* `scripts/misc_checks.py` which checks for the following things:
+Currently, following tox environments are available:
+
+* Unit tests using https://www.pytest.org[pytest]:
```
Comment (The-Compiler) [inline CONTRIBUTING.asciidoc]: I know it's my text, but I think "Tests using ..." would make more sense, as they aren't only unittests.

**Comment by flv0** [comment]
Adapted. Also tell me if you have more relevant invocations for the examples section at hand which you maybe commonly use.

**Comment by The-Compiler** [comment]
Thanks!

**Pull request merged**